src/window.cpp

```diff
@@ -1178,6 +1178,18 @@ Window *FindWindowByClass(WindowClass cls)
 	return nullptr;
 }
 
+/**
+ * Get the main window, i.e. FindWindowById(WC_MAIN_WINDOW, 0).
+ * This function asserts the main window is available.
```

Comment:
missing `if` I think?

Reply:
What about `@note If the main window is not available, this function will trigger an assert.`? Maybe without the `@note`?

I agree it's difficult to convey that it behaves slightly differently than just `FindWindowById(WC_MAIN_WINDOW, 0)`, but for a good reason: now you know it ain't `nullptr` and otherwise you don't.

Reply:
One could argue it should return `Window &`, and not have this comment at all :P I am fine with your suggestion, with or without `@note`.